test/python/edge/test.ts

```diff
@@ -4,7 +4,7 @@ import { QueryableStack, TestDriver } from "../../test-helper";
 import * as path from "path";
 import * as fs from "fs-extra";
 
-describe("full integration test", () => {
+describe.skip("full integration test", () => {
```

Comment:
I'd like to see a follow-up task to re-enable this after upgrading JSII to not forget to re-enable this 😇